packages/xo-web/src/common/xo/index.js

```diff
@@ -1238,14 +1238,23 @@ export const migrateVm = (vm, host) =>
   confirm({
     title: _('migrateVmModalTitle'),
     body: <MigrateVmModalBody vm={vm} host={host} />,
-  }).then(params => {
+  }).then(async params => {
     if (!params.targetHost) {
       return error(
         _('migrateVmNoTargetHost'),
         _('migrateVmNoTargetHostMessage')
       )
     }
-    return _call('vm.migrate', { vm: vm.id, ...params })
+
+    return _call('vm.migrate', { vm: vm.id, ...params }).catch(err =>
+      confirm({
+        icon: 'alarm',
+        title: _('forceVmMigrateModalTitle'),
+        body: _('forceVmMigrateModalMessage', {
```

Comment:
IMHO, is not a good idea to display the error in the same line than the information message. If the error don't have a message it will break the line.